asset/cloud-client/src/main/java/com/example/asset/SearchAllIamPoliciesExample.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.asset;
+
+import com.google.cloud.asset.v1p1beta1.AssetServiceClient;
+import com.google.cloud.asset.v1p1beta1.AssetServiceClient.SearchAllIamPoliciesPage;
+import com.google.cloud.asset.v1p1beta1.AssetServiceClient.SearchAllIamPoliciesPagedResponse;
+import java.util.List;
+
+// [START asset_quickstart_search_all_iam_policies]
+public class SearchAllIamPoliciesExample {
+  /**
+   * Search Iam Policies that matches the given {@code scope}, {@code query}. It only return the
+   * IAM Policies, when you have searchAllIamPolicies permission (can be granted by "Cloud Asset
+   * Viewer" role) on the given {@code scope}.
+   */
+  public static void searchAllIamPolicies(String scope, String query, List<String> assetTypes)
+      throws Exception {
+    // String scope = "projects/123456789";
```

Comment:
Do these special strings have methods to create them? Or add more explanation on how to create them.